### ISSUE TITLE
MVP2 'what may increase activity of human DHODH'... we really need better EPC

### ISSUE THREAD

**Issue by gglusman** [NCATSTranslator/Feedback#916]
MVP2, [what drugs may increase expression of human DHODH](https://ui.test.transltr.io/main/results?l=DHODH%20(Human)&i=NCBIGene:1723&t=1&r=0&q=f21ff5a7-8fd6-4d6d-b463-b88ba6596487):

The top answer (score=5) is [5'-(Cyclopropylmethyl)spiro[cyclohexane-1,1'-furo[3,4-c]pyridine]-3',4'-dione](https://pubchem.ncbi.nlm.nih.gov/compound/162673212). This is a lookup edge:
![image](https://github.com/user-attachments/assets/5e47b44b-1a97-4ab3-a6dc-ee5447f08822), supported by [one paper](http://www.ncbi.nlm.nih.gov/pubmed/31740051) and one source, BindingDB.

I looked into the full text of the paper and couldn't really find mention of the molecule. Various related and similar things are mentioned, but I couldn't clearly match it to this molecule.

The EPC link to BindingDB sends me to [the wiki page](https://github.com/NCATSTranslator/Translator-All/wiki/bindingdb), from which I went to [BindingDB's homepage](https://www.bindingdb.org/), where I searched for the molecule by name... unsuccessfully. So I used its ChEMBL identifier instead (CHEMBL4795021) - it was offered as a synonym by [the PubChem page](https://pubchem.ncbi.nlm.nih.gov/compound/162673212), as linked from the result in the UI.
The BindingDB page indeed claims that this compound targets human DHODH:
![image](https://github.com/user-attachments/assets/a007e901-2071-4b1b-a787-5fa67e2d85f7)
The evidence it provides for this assertion is "Curated by ChEMBL", with a link to ChEMBL's homepage.
So I went to ChEMBL and searched for this compound (easy given I have the ChEMBL identifier). In the [compound report card](https://www.ebi.ac.uk/chembl/compound_report_card/CHEMBL4795021/), I scroll down to the "Target Predictions" section, where I find a small table with 339 predicted targets. Among them, I see:
![image](https://github.com/user-attachments/assets/25e13c48-1c5c-47a2-8b74-7bba226d1b08)
Since the content of the table is rather obscure (three levels of % confidence, and an unclear Activity Threshold), I went to [the suggested paper](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-018-0325-4) to educate myself. In the Data Preparation section of the paper, I read that, apparently, there is a "default activity threshold of 6.5 logarithmic value units" for classifying things into active or inactive, which this prediction doesn't meet... so I probably shouldn't give much credence to things labeled 'Inactive'.

So, in summary, after much digging through obscure EPC rabbit holes, the top results seems to be based on a weak prediction based on results from just one paper - that I'm still not sure even talks about this molecule.

**Comment by gglusman** [comment]
Digging via ARAX interface, I see this from Unsecret:
![image](https://github.com/user-attachments/assets/f8e412a4-6aa4-4069-8bd6-2b360ca2cb95)
The two support edges shown appear to be identical.

From Aragorn:
![image](https://github.com/user-attachments/assets/16e751e3-2828-4b3c-9566-cf94a2dde57c)
![image](https://github.com/user-attachments/assets/29220bf4-474e-499f-bc1b-978ed83e3bfe)

So it looks like the two ARA assertions are based on exactly the same source information.

**Comment by gglusman** [comment]
Just to make more explicit a point that may have been a bit obscure above. BindingDB is tagged in the EPC as primary_knowledge_source, but it in fact points at ChEMBL for its source. It is not entirely clear to me whether ChEMBL is the primary source, either...

**Comment by sstemann** [comment]
tagging Unsecret and EPC for their review of the findings, which are thorough and much appreciated.
